mscore/pathlistdialog.cpp

```diff
@@ -41,8 +42,12 @@ PathListDialog::PathListDialog(QWidget* parent)
 
 void PathListDialog::addClicked()
       {
-      QString newPath = QFileDialog::getExistingDirectory (this, tr("Choose a directory"),
-         QString("%1/%2").arg(QDesktopServices::storageLocation(QDesktopServices::DocumentsLocation)).arg(QCoreApplication::applicationName()));
+      QString newPath = QFileDialog::getExistingDirectory(
+         this,
+         tr("Choose a directory"),
+         QString("%1/%2").arg(QDesktopServices::storageLocation(QDesktopServices::DocumentsLocation)).arg(QCoreApplication::applicationName()),
+         preferences.nativeDialogs ? QFileDialog::Options() : QFileDialog::DontUseNativeDialog
```

Comment:
ok, I understand what you're saying.  so is it better to have a big if statement for all of these?